src/tap/include/tap/MakeTapcell.h

```diff
@@ -45,14 +44,11 @@ namespace ord {
 
 class OpenRoad;
 
-tap::Tapcell *
-makeTapcell();
+tap::Tapcell* makeTapcell();
 
-void
-deleteTapcell(tap::Tapcell *tapcell);
+void deleteTapcell(tap::Tapcell* tapcell);
 
-void
-initTapcell(OpenRoad *openroad);
+void initTapcell(OpenRoad* openroad);
 
-} // namespace
+}  // namespace ord
 #endif
```

Comment:
Add a new line after `#endif`

Reply:
There are already one new line after #endif and some of the other files as well. Do I add another or should there just be one more line at the end of each file?